executor.go

```diff
@@ -160,6 +161,12 @@ func (e *executor) runJob(f jobFunction) {
 		if lockKey == "" {
 			lockKey = f.funcName
 		}
+		if e.distributedElection != nil {
```

Comment:
would it be problematic if someone runs both election and the locker? They seem mutually exclusive in that you'd use one or the other, so perhaps this should be an if/else?

Reply:
done. 😁 Can you see if this can be changed ?